topic-auto-create/README.md

```diff
@@ -0,0 +1,134 @@
+        "database.user": "postgres",
+        "database.password": "postgres",
+        "database.dbname" : "postgres",
+        "database.server.name": "dbserver1",
+        "schema.include.list": "inventory",
+
+        "topic.creation.default.replication.factor": 1,
+        "topic.creation.default.partitions": 5,
+        "topic.creation.default.cleanup.policy": "compact",
+        "topic.creation.default.compression.type": "gzip",
+        "topic.creation.default.delete.retention.ms" : 2592000000,
+
+        "topic.creation.groups": "productlog",
+
+        "topic.creation.productlog.include": "dbserver1\\.inventory\\.product.*",
+        "topic.creation.productlog.replication.factor": 1,
+        "topic.creation.productlog.partitions": 10,
+        "topic.creation.productlog.cleanup.policy": "delete",
+        "topic.creation.productlog.compression.type": "producer",
+        "topic.creation.productlog.retention.ms": 7776000000
+    }
+}
+```
+
+### Sending connector config
+
+```bash
+$ curl -X POST -H "Accept: application/json" -H "Content-Type: application/json" -d @connector.json "http://localhost:8083/connectors" | json_pp
+
+% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  2154  100  1009  100  1145   1318   1496 --:--:-- --:--:-- --:--:--  2815
+{
+   "config" : {
+      "connector.class" : "io.debezium.connector.postgresql.PostgresConnector",
+      "database.dbname" : "postgres",
+      "database.hostname" : "postgres",
+      "database.password" : "postgres",
+      "database.port" : "5432",
+      "database.server.name" : "dbserver1",
+      "database.user" : "postgres",
+      "name" : "inventory-connector",
+      "schema.include.list" : "inventory",
+      "tasks.max" : "1",
+      "topic.creation.default.cleanup.policy" : "compact",
+      "topic.creation.default.compression.type" : "gzip",
+      "topic.creation.default.delete.retention.ms" : "2592000000",
+      "topic.creation.default.partitions" : "5",
+      "topic.creation.default.replication.factor" : "1",
+      "topic.creation.groups" : "productlog",
+      "topic.creation.productlog.cleanup.policy" : "delete",
+      "topic.creation.productlog.compression.type" : "producer",
+      "topic.creation.productlog.retention.ms" : "7776000000",
+      "topic.creation.productlog.include" : "dbserver1\\.inventory\\.product.*",
+      "topic.creation.productlog.partitions" : "10",
+      "topic.creation.productlog.replication.factor" : "1"
+   },
+   "name" : "inventory-connector",
+   "tasks" : [],
+   "type" : "source"
+}
+``` 
+
+### Validating the result
+
+```bash
+## Login into the Debezium container
+$ docker exec -it topic-auto-create_kafka_1 bash
+
+## Check the orders topic, if the `default` group config was applied
+$ /kafka/bin/kafka-topics.sh --bootstrap-server $HOSTNAME:9092 --describe --topic dbserver1.inventory.orders
```

Comment:
I'd probably have used the Debezium tooling image and kafkacat, but it's fine that way, too.